src/plugins/VuePlugin.ts

```diff
@@ -74,6 +80,23 @@ function toFunction (code) {
   return vueTranspiler('function render () {' + code + '}')
 }
 
+function compileTemplateContent (context: any, engine: string, content: string) {
+    const cons = require('consolidate');
```

Comment:
Shouldn't this be installed as well? Maybe we could mention it in the docs as well